ToolJet/ToolJet
# feature/translate spanish document

- Hide user data in group permissions get user API (#4141)
- Bump to v1.26.1
- [ hotfix ]  bug fixed : general tooltip is not displaying for button widget (#4152)
- [ hotfix ] : [ Table Widget ]  filter pop-up and dropdown inside it are  not scrollable (#4098)
- [docs] ClickHouse plugin (#3951)
- Fixed bug - default marker color value for the chart widget is added (#4306)
- [docs] rename Workspace Environment Variables to Workspace Variables (#4360)
- Updated default text color for table widget (#4247)
- Adds Italian translation (#4328)
- [ Enhancement ] : [ Table widget ] added support to download data in excel (#4134)
- fix: shell variable initialization
- exposed updatedDate (#4263)
- throw error with response body instead of response (#4365)
- [Improvements] Table: Adds Support for Server side filtering (#4349)
- [docs] Update architecture.md (#4368)
- Added border color property to button styles (#4356)
- Feature/list view border color (#4300)
- [docs] Fix typos and grammatical errors in adding-widget.md (#4366)
- Update hotfix version
- [feature]:: Mariadb plugin (#4090)
- all: fix spelling (#4326)
- Added Border color  style property to Container widgets (#4202)
- Add docs for ECS (#4370)
- [docs] Fix spelling and fix whitespace in Markdown (#4377)
- Translated component manager in french (#4371)
- Add Indonesian localization (#4287)
- Hide Apps tab for Admin (#2634) (#4221)
- Feature/automatic columns generation (Task ID: CU-2vjwn7r) (#4149)
- fixes: exposes the latest currentData in the table (#4364)
- [docs] Fix Redirect error (#4395)
- [localization] translate strings (editor & header) section in Indonesian (#4308)
- Show user list for all groups other than all users (#4398)
- Feature :: Table - option to hide columns (#4284)
- Add optional chaining operator to avoid crash while reading autogenerateColumns property
- [ Feature ] :: Table :: event on discard changes :: (#4158)
- Feature :: Table sort disable/enable (#4142)
- Enhancements :: Table bugfixes and option for pagination number added (#4083)
- [ Feature ] :: Reorder Table columns using drag and drop (#4327)
- [ Improvement ] :: Table  widget (#4092)
- [Bugfix] fix filter change with select component and scroll options with multiple filters (#4404)
- Fixed the buttons default text color to #000 (#4361)
- Bugfix  :: Table sort not visible (#4408)
- Fix :: user group table ui breaking  (#4418)
- fix:: resizer not visible in preview mode (#4427)
- fix :: converting data as per current code implementaion , xl dowload not working (#4424)
- [hotfix] Fixed oauth refresh token issue (#4406)
- [Chore] widget fixes (#4411)
- fix  :: automatic state reset for hiding columns on data change restricted (#4429)
- updates the exposed vars (#4426)
- [Bugfix] show or hide pagination footer only if client or server side pagination is enabled (#4430)
- revert :: style change (#4431)
- Triggered selection only when it is dragged on main canvas (#4342)
- Update hotfix version
- mariadb and improvement docs (#4432)
- Bump version
- Add happy path spec for password input widget (#4112)
- hotfix :: sort issue table (#4436)
- Bump hotfix version
-  Make environment variable's delete modal's color in dark mode (#4466)
- Error Message on User Removal (#4464)
- Trim version name input (#4462)
- Error toast should be at center in profile settings (#4471)
- Fixed error message on removal of app from group. (#4458)
- [Fix] Delete app not working if any user mentioned in comments (#4161)
- bug fixed : table widget is not consistent with the dark theme (#3987)
- Changed "on sorting columns" label in table (#4500)
- Adds default highlight color (#4498)
- [localization] Translate global strings in spanish (#4230)
- Fix tooljet DOM dittachment issue (#4468)
- Fix spelling (#4400)
- [localization] Translate Profile Settings Page in French (#4185) (#4359)
- Rename Environment Variables to Workspace Variables (#4307)
- Reset newGroupName to null after creating new group (#4294)
- On Change added to NumberInput (#4198)
- [localization] Translate login and signup page in Spanish #4180 (#4324)
- [localization] Created the it file translation (#4259)
- [docs] improve docker documentation (#4508)
- bug fixed : selecting badge is breaking the table (#4416)
- bug fixed : type error (#4445)
- fix: disable multiple selection of components while comments feature is toggled on (#4470)
- Chore: Setup preview apps with Render (#4520)
- update render.yaml
- Renamed "Bulk Update" event to "Save changes" in table (#4516)
- Add RU localisation for Tooljet (#4465)
- update render.yaml
- Translated App Card in French (#4219)
- Chore: Update preview app expiry (#4522)
- [localization] Translate Profile Settings Page in Spanish #4186 (#4205)
- [localization] Translate Component Manager in Spanish (#4521)
- [ Hotfix ] :: App breaking while reconnecting sheets (#4514)
- Bump to v1.27.2
- merge main to develop (#4527)
- [Bug]: Fixed Bug on Query Panel Height (#4511)
- Disabled picking browsers language in localisation (#4502)
- Fixed bug on displaying newly added data source (#4473)
- Bump hotfix version
- Codehinter optimisations (#4537)
- Bump patch version
- Added check while fetching the queryPanel height from localStorage (#4543)
- Bump hotfix version
- Initial commit, translate first texts
